src/clr-angular/popover/tooltip/tooltip-trigger.ts

```diff
@@ -1,14 +1,34 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { Directive, HostListener } from '@angular/core';
+import { Directive, HostListener, Input } from '@angular/core';
 import { IfOpenService } from '../../utils/conditional/if-open.service';
+import { TooltipIdService } from './providers/tooltip-id.service';
+import { Subscription } from 'rxjs';
 
-@Directive({ selector: '[clrTooltipTrigger]', host: { '[attr.tabindex]': '0', '[class.tooltip-trigger]': 'true' } })
+@Directive({
+  selector: '[clrTooltipTrigger]',
+  host: {
+    tabindex: '0',
+    '[class.tooltip-trigger]': 'true',
+    '[attr.aria-describedby]': 'ariaDescribedBy',
+    '[attr.aria-label]': 'tooltipLabel',
+    role: 'button',
```

Comment:
Isn't `role: 'tooltip'` a better fit for this?

Reply:
Not for the trigger. tooltip needs to be on the content.

Reply:
Ah I see.